InfiniteLoopers-App/src/main/java/com/isa/account/LoginManager.java

```diff
@@ -1,30 +1,71 @@
 package com.isa.account;
 
+import java.util.List;
+
+import static com.isa.menu.Menu.menuInvoke;
+
 public class LoginManager {
-    private AuthenticationService authService;
-    private UserInputReader userInputReader;
+    public LoggedUser getLoggedUser() {
+        return loggedUser;
+    }
+
+    private LoggedUser loggedUser = new LoggedUser();
+    private static AuthenticationService authService;
+    private static UserInputReader userInputReader;
 
     public LoginManager(AuthenticationService authService) {
         this.authService = authService;
         this.userInputReader = new UserInputReader();
     }
 
-    public void login () { //TODO zapytac czy ma byc void
+    public void login() {
         System.out.println("Witam. Prosze wprowadzic login i haslo.");
 
-        String userName = userInputReader.readNonEmptyString("Login: ");
+        String loginUser = userInputReader.readNonEmptyString("Login: ");
         String passwordUser = userInputReader.readNonEmptyString("Haslo: ");
 
-        User user = authService.login(userName, passwordUser);
+        User user = authService.login(loginUser, passwordUser);
 
-        if (user != null) {
-            new LoggedUser().logUser(user);
+        if (user.getUserRole() == null){
+            System.out.println("Twoje konto nie jest jeszcze aktywne. Sprobuj ponownie pozniej lub skontaktuj sie z administratorem.");
+            menuInvoke();
+        }
+        if (user != null  ) {
```

Comment:
to będzie zawsze różne od nulla, ewentualnie wcześniej nam poleci NPE, bo na potencjalnym nullu sprawdzasz user role, więc jeśli tam nam nie poleci błąd to tutaj to nigdy nie będzie null